### PR TITLE
Create filter bar

### LINKED ISSUE #11: Make filter bar
Create filter bar

### PR DESCRIPTION
**Pull request by lindacnatt**
filter bar + OL icon in head

fix #11

### CODE REVIEW

File: src/atoms/FilterButton.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const FilterButt = styled.div<FilterButtProps>`
+  background: ${props => {
+    return props.background;
+  }};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  max-width: 10%;
+  min-width: 20%;
+  color: black;
```
Comment (johannesloor) [inline src/atoms/FilterButton.tsx]: Maybe use another color? A bit hard to read sometimes

File: src/atoms/FilterButton.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    -o-transition: height 0.35s;
+    -webkit-transition: height 0.35s;
+    transition: height 0.35s;
+    height: 2.5vw;
+    overflow: hidden;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.sm + 'px'}) {
+    height: 10vw;
+    color: black;
+    font-size: 2.5vw;
```
Comment (johannesloor) [inline src/atoms/FilterButton.tsx]: Make a bit smaller? Could not read "underhållning" when pressed on mobile

File: src/molecules/header.tsx [hunk 3]
```diff
@@ -28,14 +29,20 @@ const HeaderWrapper = styled.div`
...
+    padding-bottom: 10%;
   }
 `;
 
 const Logo = styled(Img)`
-  width: 100%;
+  width: 90%;
+  @media (max-width: ${({ theme }) => theme.breakpoints.sm + 'px'}) {
+    width: 90%;
+    display: none;
```
Comment (johannesloor) [inline src/molecules/header.tsx]: Maybe the filterbar should stick to the top on scrolling instead? It would be nice to see the logo at first

Comment (lindacnatt) [reply src/molecules/header.tsx]: The filter bar cannot be in the header if only the filterbar should stick to the top when scrolling. Should I remove it from the header?

**Pull request merged**